### PR TITLE
add tolerations and priority class name to network-mapper daemon set

### PR DESCRIPTION
**Pull request by romga**
### Description
Adding the option to add tolerations and priority class name to network mapper daemon set, so the network mapper could be added to the entire nodes in the cluster
### References

Include any links supporting this change such as a:

- GitHub Issue/PR number addressed or fixed
https://github.com/otterize/helm-charts/issues/92

### Checklist

- [  ] I have added documentation for new/changed functionality in this PR and in github.com/otterize/docs

### CODE REVIEW

**Comment by romga** [comment]
recheck

**Comment by romg67** [comment]
I have read and understood the CLA and hereby agree to its terms by making this Pull Request Comment.

**Comment by romga** [comment]
recheck

**Comment by romga** [comment]
I have read and understood the CLA and hereby agree to its terms by making this Pull Request Comment.

File: network-mapper/README.md [hunk 1]
```diff
@@ -22,7 +22,9 @@
 | `sniffer.pullPolicy`       | Sniffer pull policy.       | `(none)`                 |
 | `sniffer.pullSecrets`      | Sniffer pull secrets.      | `(none)`                 |
 | `sniffer.resources`        | Resources override.        | `(none)`                 |   
-
+| `sniffer.resources`        | Resources override.        | `(none)`                 |
+| `sniffer.tolerations`      | Tolerations ovveride.      | `(none)`                 |
```
Comment (evyatarmeged) [inline network-mapper/README.md:26]:
```suggestion
| `sniffer.tolerations`      | Tolerations override.      | `(none)`                 |   
```

**Review by evyatarmeged — APPROVED** [review]

**Comment by orishoshan** [comment]
Documentation added in https://github.com/otterize/docs/pull/85.
Closes #93 .

**Pull request merged**